src/descriptive_names.yml

```
@@ -246,6 +246,146 @@ structures:
     reserve_down: "Resources used as reserve for removing capacity"
     data: "Additional data"
 
+  CycleLife:
+    stack_cost: "Relative cost for replacing a battery stack"
+
+  ## EnergyModelsHeat
+  HeatPump:
+    cap:                    "Installed capacity"
+    t_source:               "Temperature profile of the heat source"
+    t_sink:                 "Sink temperature of the condensator"
+    eff_carnot:             "Carnot Efficiency"
+    opex_var:               "Variable operating expense per energy unit produced"
+    opex_fixed:             "Fixed operating expense per installed capacity"
+  ResourceHeat:
+    t_supply:               "Supply temperature in °C"
+    t_return:               "Return temperature in °C"
+  DHPipe:
+    cap:                    "Heat transport capacity of the pipe"
+    t_ground:               "Ground temperature in °C"
+  HeatExchanger:
+    cap:                    "Installed capacity"
+    opex_var:               "Variable operating expense per energy unit produced"
+    opex_fixed:             "Fixed operating expense"
```

```suggestion
    opex_fixed:             "Fixed operating expense per installed capacity"
```

It uses the base functionality.